lib/samples/display_web_scene_from_portal_item/display_web_scene_from_portal_item.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/common/common.dart';
+import 'package:flutter/material.dart';
+
+class DisplayWebSceneFromPortalItem extends StatefulWidget {
+  const DisplayWebSceneFromPortalItem({super.key});
+
+  @override
+  State<DisplayWebSceneFromPortalItem> createState() =>
+      _DisplayWebSceneFromPortalItemState();
+}
+
+class _DisplayWebSceneFromPortalItemState
+    extends State<DisplayWebSceneFromPortalItem>
+    with SampleStateSupport {
+  // Create a controller for the scene view.
+  final _sceneViewController = ArcGISSceneView.createController();
+  // A flag for when the scene view is ready and controls can be used.
+  var _ready = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
```

Comment:
We don't need a SafeArea because there are no controls at the bottom. We want edge-to-edge.